packages/strapi-plugin-versioning/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "strapi-plugin-versioning",
+  "version": "0.1.0",
+  "description": "Strapi plugin for collecting versions of content type entities",
+  "main": "src/index.ts",
+  "scripts": {
+    "test": "jest --coverage",
+    "build": "tsc",
+    "clean": "rimraf dist coverage",
+    "clean-all": "rimraf dist coverage node_modules",
+    "set-development-package-json": "ts-node ../../scripts/set-development-package-json.ts",
+    "set-publish-package-json": "ts-node ../../scripts/set-publish-package-json.ts",
+    "extract-dist-to-root": "ts-node ../../scripts/extract-dist-to-root.ts",
+    "clean-dist-from-root": "ts-node ../../scripts/clean-dist-from-root.ts",
+    "prepublishOnly": "yarn build && yarn set-publish-package-json",
+    "postpublish": "yarn set-development-package-json",
+    "postinstall": "cp -r dist/extension-file-to-copy/content-manager $INIT_CWD/extensions"
+  },
+  "dependencies": {
+    "lodash": "^4.17.12",
+    "strapi-utils": "^3.0.6"
```

Comment:
is it compatible with strapi in general? (3.1.4?)

Reply:
it is

Reply:
maybe it is, but they doesn't guarantee, they doesn't have independent packages, and if Strapi is 3.1.4 then strapi utils also should be 3.1.4